src/components/HelpTopicLink/HelpTopicLink.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+type HelpTopicLinkProps = {
+  topicId: string;
+  buttonVariant?: ButtonVariant;
+};
+
+const HelpTopicLink: React.FC<HelpTopicLinkProps> = ({ topicId, children, buttonVariant = ButtonVariant.link }) => {
+  const {
+    helpTopics: { setActiveTopic, enableTopics, disableTopics },
+  } = useChrome();
+
+  React.useEffect(() => {
+    enableTopics(topicId);
+
+    return () => disableTopics(topicId);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Thinking out loud because I had to do this in my PR too.  Wondering if there is a lint rule that will allow you to have an empty array without completely disabling the exhaustive-deps rule.   Don't expect a change in this PR, just something to think about for the future.